src/bin/dual-iir.rs

```diff
@@ -201,7 +221,15 @@ const APP: () = {
             network,
             digital_inputs: stabilizer.digital_inputs,
             telemetry: TelemetryBuffer::default(),
-            settings: Settings::default(),
```

Comment:
I'd keep this (`Settings::default()`) inlined.

Reply:
We use the settings below in the construction of the signal generator, which is why it was broken out. Otherwise, we'd have to construct multiple default settings.

Reply:
Ah. I overlooked that. Ack then.